modules/images/regenerate-existing-images/background-process/class-perflab-background-job.php

```diff
@@ -0,0 +1,313 @@
+<?php
+/**
+ * Background job class.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Class Perflab_Background_Job.
+ *
+ * Manage and run the background jobs.
+ *
+ * Following fields are being stored for a background job.
```

Comment:
Per [Number order description](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#description), Use a colon instead of a period.
```suggestion
 * Following fields are being stored for a background job:
```